### PR TITLE
Feature/cv carlos-calvo

### PR DESCRIPTION
**Pull request by carlosclatg**

### CODE REVIEW

File: staff/carlos-calvo/cv/index.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+            <li>Name: Carlos</li>
+            <li>Surname: Calvo</li>
+            <li>Birth Date: 09/12/1986</li>
+            <li>mail: carloscl1986@gmail.com</li>
+            <li>Mobile Phone: 620092996</li>
+        </ul>
+    </section>
+    <hr/>
+    <section>
+        <h1><b>FORMATION</b></h1>
```
Comment (nicopixel) [inline staff/carlos-calvo/cv/index.html]: You can add every h1 from every section inside a header element

File: staff/carlos-calvo/cv/index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        <ul>
+            <li>Industrial Engineering
+                <ul>
+                    <li>UPC, Barcelona</li>
+                    <li>2004 - 2010</li>
```
Comment (nicopixel) [inline staff/carlos-calvo/cv/index.html]: Also, you can use the `<datetime>` tag to wrap the date & times:
`<time datetime="2004-01-01">2004</time> - <time datetime="2010-12-31">2010</time>`

File: staff/carlos-calvo/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV Carlos Calvo</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
```
Comment (nicopixel) [inline staff/carlos-calvo/cv/index.html]: without styles

File: staff/carlos-calvo/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    <script src="main.js"></script>
```
Comment (nicopixel) [inline staff/carlos-calvo/cv/index.html]: without any javascript please

Comment (gcelma) [inline staff/carlos-calvo/cv/index.html]: I would delete the links to css and js

File: staff/carlos-calvo/cv/index.html (hunk 1 above)
Comment (carlosclatg) [reply staff/carlos-calvo/cv/index.html]: OK! Nice :)

File: staff/carlos-calvo/cv/index.html (hunk 2 above)
Comment (carlosclatg) [reply staff/carlos-calvo/cv/index.html]: OK, that's better

File: staff/carlos-calvo/cv/index.html (hunk 4 above)
Comment (carlosclatg) [reply staff/carlos-calvo/cv/index.html]: Deleted

File: staff/carlos-calvo/cv/index.html (hunk 1 above)
Comment (nicopixel) [reply staff/carlos-calvo/cv/index.html]: Sorry Carlos, I've asked to Manu and he told me that when you have only one heading you can remove the `<heading>` element.
Sorry for the bad correction

**Pull request merged**